app/src/main/java/de/tum/in/tumcampusapp/component/ui/cafeteria/CafeteriaMenuViewHolder.kt

```diff
@@ -12,6 +12,7 @@ import de.tum.`in`.tumcampusapp.component.ui.cafeteria.model.CafeteriaWithMenus
 import de.tum.`in`.tumcampusapp.component.ui.overview.CardInteractionListener
 import de.tum.`in`.tumcampusapp.component.ui.overview.card.CardViewHolder
 import org.joda.time.format.DateTimeFormat
+import java.util.*
```

Comment:
I just saw you @CommanderStorm mentioning in another PR that you want explicit more than implicit? For me that import is fine, but just to assure

Reply:
Actually, for java.utils.* we even have an exception in our linting suite, so this one is probably fine. Also android studio collapses this one, if I try to expand it :)